src/seeds.gratitude.cpp

```diff
@@ -28,11 +28,33 @@ ACTION gratitude::reset () {
   stats.emplace(_self, [&](auto& item) {
     item.round_id = 1;
     item.num_transfers = 0;
+    item.num_acks = 0;
     item.volume = asset(0, gratitude_symbol);
+    item.round_pot = asset(0, seeds_symbol);
   });
 
 }
 
+ACTION gratitude::migratestats() {
+  require_auth(get_self());
+
+  name old = "gratz.seeds"_n;
```

Comment:
Not sure how this migration is supposed to work?

did you change the table name?

Reply:
Fixed